arangod/V8Server/V8DealerFeature.cpp

```diff
@@ -1251,17 +1251,16 @@ V8Context* V8DealerFeature::enterContext(
     TRI_ASSERT(context != nullptr);
 
     _idleContexts.pop_back();
+    TRI_ASSERT(context != nullptr);
```

Comment:
Do we neet to assert again here? See line 1251

Reply:
No, I just move from old line

Reply:
I'd say we need to assert before TRACE log at 1249 as it touches context

Reply:
true